### PR TITLE
Added Swedish Translations

### PR DESCRIPTION
**Pull request by FelixZY**
I have translated almost all strings but am still unsure about a few and could use some help putting them into context (see comment section below).

### CODE REVIEW

File: sv-SE.lproj/Menu.strings [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+/* Typora Menu */
+
+"About Typora" = "Om Typora";
+"Preferences" = "Inställningar";
+"Check for Updates" = "Sök efter uppdateringar";
+"Services" = "Tjänster";
+"Hide Typora" = "Göm Typora";
+"Hide Others" = "Göm andra"; /* Context? */
+"Show All" = "Visa alla"; /* Context? "Visa allt" may be a better translation depending on context */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: They are on macOS, follow the standard transalation is OK ->  https://github.com/martnst/localize-mainmenu/blob/master/languages/sv.json

File: sv-SE.lproj/Menu.strings [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+"Redo" = "Gör om";
+"Cut" = "Klipp ut";
+"Copy" = "Kopiera";
+"Paste" = "Klistra in";
+"Copy as Markdown" = "Kopiera som Markdown";
+"Copy as HTML Code" = "Kopiera som HTML";
+"Paste as Plain Text" = "Klistra in utan formattering";
+"Select All" = "Välj allt";
+"Select Line/Sentence" = "Välj rad/mening";
+"Select Styled Scope" = "Välj Styled Scope"; /* What is meant by a 'styled scope'? /*
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: Example:

> This is a text **with strong style**

When user move cursor inside "with strong style", and click this menu item, all test between "**" ("with strong style") will be selected.

File: sv-SE.lproj/Menu.strings [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+"Jump to Bottom" = "Gå till botten";
+"Math Tools" = "Matteverktyg";
+"Refresh All Math Expressions" = "Ladda om alla matteuttryck";
+"Image Tools" = "Bildverktyg";
+"Insert Local Images" = "Infoga lokala bilder";
+"Upload Local Images via iPic" = "Ladda upp lokala bilder via iPic";
+"When Insert Local Image" = "När en lokal bild infogas";
+"Copy Image File to Folder" = "Kopiera bildfil till mapp";
+"Upload Image via iPic" = "Ladda upp bild via iPic";
+"Use Image Root Path" = "Använd mapp för bilder"; /* Root path for images associated with a markdown file, correct? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: yes

File: sv-SE.lproj/Menu.strings [hunk 4]
```diff
@@ -0,0 +1,258 @@
...
+"Replace Next" = "Ersätt nästa";
+"Spelling and Grammar" = "Stavning och grammatik";
+"Check Document Now" = "Kontrollera dokumentet nu";
+"Check Spelling While Typing" = "Kontrollera stavning under skrift";
+"Check Grammar With Spelling" = "Kontrollera grammatik under skrift";
+"Correct Spelling Automatically" = "Korrigera stavning automatiskt";
+"Substitutions" = "Substitutioner"; /* Context? */
+"Smart Quotes" = "Smarta citattecken";
+"Smart Dashes" = "Smarta streck";
+"Text Replacement" = "Textersättning"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: still https://github.com/martnst/localize-mainmenu/blob/master/languages/sv.json

File: sv-SE.lproj/Menu.strings [hunk 5]
```diff
@@ -0,0 +1,258 @@
...
+"Find Previous" = "Sök föregående";
+"Find and Replace" = "Sök och ersätt";
+"Replace" = "Ersätt";
+"Replace Next" = "Ersätt nästa";
+"Spelling and Grammar" = "Stavning och grammatik";
+"Check Document Now" = "Kontrollera dokumentet nu";
+"Check Spelling While Typing" = "Kontrollera stavning under skrift";
+"Check Grammar With Spelling" = "Kontrollera grammatik under skrift";
+"Correct Spelling Automatically" = "Korrigera stavning automatiskt";
+"Substitutions" = "Substitutioner"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: still https://github.com/martnst/localize-mainmenu/blob/master/languages/sv.json

File: sv-SE.lproj/Menu.strings [hunk 6]
```diff
@@ -0,0 +1,258 @@
...
+
+
+/* Themes Menu */
+"Themes" = "Teman";
+
+/* Window Menu */
+"Window" = "Fönster";
+"Minimize" = "Minimera";
+"Zoom" = "Zooma";
+"Bring All to Front" = "Bring All to Front"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: https://github.com/martnst/localize-mainmenu/blob/master/languages/sv.json

File: sv-SE.lproj/Menu.strings [hunk 7]
```diff
@@ -0,0 +1,258 @@
...
+"Reveal in Sidebar" = "Visa i sidopanel";
+"Open in New Window" = "Öppna i nytt fönster";
+"Copy File Path" = "Kopiera filsökväg";
+"New File" = "Ny fil";
+"Rename" = "Byt namn";
+"New Folder" = "Ny mapp";
+"Move to Trash" = "Flytta till papperskorgen";
+
+/* Electron */
+"Learn Spelling" = "Learn Spelling"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: see https://user-images.githubusercontent.com/587790/27765620-7290c3b8-5e6b-11e7-83c3-3245459f195c.png

File: sv-SE.lproj/Menu.strings [hunk 8]
```diff
@@ -0,0 +1,258 @@
...
+"Copy / Paste As…" = "Kopiera / Klistra in som…";
+"Math" = "Matte";
+"Insert" = "Infoga";
+"Inspect Element" = "Inspektera element";
+"Copy to MS Word" = "Kopiera till MS Word";
+"Paragraph (before)" = "Stycke (före)"; /* Context? */
+"Paragraph (after)" = "Stycke (efter)"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: image

Insert paragraph before or after current block

File: sv-SE.lproj/Panel.strings [hunk 9]
```diff
@@ -0,0 +1,247 @@
...
+/* General Panel */
+"General" = "Allmänt";
+"Window Style" = "Fönsterstil";
+"Classic" = "Klassisk";
+"Seamless" = "Sömlös";
+"Seamless window only support OS X later than 10.10." = "Sömlöst fönster stöds endast av OS X nyare än 10.10.";
+"Themes" = "Teman";
+"Open Theme Folder" = "Öppna temamapp";
+"Please check `help`->`Quick Start` for short introduction. Visit http://theme.typora.io to get more themes or instructions." = "Gå till `hjälp`->`Snabbstart` för en kort introduktion. Besök http://theme.typora.io för att hämta fler teman eller få mer hjälp.";
+"Word Count" = "Ordräknare"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Panel.strings]: image

File: sv-SE.lproj/Panel.strings [hunk 10]
```diff
@@ -0,0 +1,247 @@
...
+
+
+"Total" = "Total";
+"Word count only support OS X later than 10.10." = "Ordräkning stöds endast av OS X nyare än 10.10.";
+"Indent Size for Code" = "Storlek på indrag för kod";
+"Auto wrap long lines" = "Radbryt långa textrader automatiskt";
+
+"Sorry, current version of Typora won't print background color.\nSo dark theme is not yet supported for printing. \n\nPlease choose a light theme or export this document to PDF." = "Tyvärr kan inte denna version av Typora skriva ut bakgrundsfärg i dokument. Ett mörkt tema kan därför inte användas för utskrifter. \nByt till ett ljust tema eller exportera dokumentet som PDF i stället.";
+"Export to PDF" = "Exportera som PDF";
+"Pretty indentation" = "Snyggt indrag"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Panel.strings]: Disabled:
image

Enabled:
image

File: sv-SE.lproj/Panel.strings [hunk 11]
```diff
@@ -0,0 +1,247 @@
...
+"Open Advanced Settings" = "Öppna avancerade inställningar";
+"Reset Advanced Settings" = "Återställ avancerade inställningar";
+
+/* Image relates on macOS */
+"This fucntion requires latest version of iPic to be installed." = "Denna funktion kräver att senaste versionen av iPic installeras."; /* That the latest version is installed or that the latest version is to be installed? */
+"Get iPic" = "Skaffa iPic";
+"Cannot Continue…" = "Kan inte fortsätta…";
+"This feature require macOS >= 10.11" = "Denna funktion kräver macOS >= 10.11";
+"One More Step" = "Ett steg till";
+"Notice" = "Notera"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Panel.strings]: Title of a **warning** dialog.

File: sv-SE.lproj/Menu.strings [hunk 12]
```diff
@@ -0,0 +1,258 @@
...
+"Privacy Policy" = "Integritetspolicy";
+"Credits" = "Credits";
+"Change Log" = "Ändringslogg";
+"Website" = "Hemsida";
+"Feedback" = "Feedback";
+"About" = "Om";
+
+/* Touchbar */
+"Block Styles" = "Block Styles"; /* Context? */
+"Inline Styles" = "Inline Styles"; /* Context? */
```
Comment (abnerlee) [inline sv-SE.lproj/Menu.strings]: There a name of a button group (or toolbar group)

The first group are responsible for changing block styles (between paragraph, h1~h5, table, etc)
The send group are responsible for changing inline styles (between normal text, strong, emphasis, code, etc)

**Comment by FelixZY** [comment]
Thank you!

I have been busy this weekend but will try to update this PR next week.

**Comment by FelixZY** [comment]
Remaining comments have been fixed. The PR should be ready to merge.

**Pull request merged**